### ISSUE TITLE
Ability to Edit Inventory Item Requests from Finance System

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#1117]
As a warehouse admin, I may need to edit an inventory item request. For example, if the user selected the wrong item, or quantity, I need to be able to quickly update that information before I issue items of the customer.

### Preconditions

- The inventory item request (transaction) must be in a state of `New Request`. Issued, Returned, or Cancelled requests should not be modified.

- To edit, an "Edit" icon will be visible in the corresponding column in the transactions table. Clicking on it will display a form in a modal, where the item, quantity, and comment can be modified.

- Any changes submitted will then be reflected in the Data Tracker as well.

![Screen Shot 2020-01-10 at 12.58.08 PM.png](https://images.zenhubusercontent.com/5caf676becad11531cc417cb/0401c5cd-3249-41d1-9bd2-b4321c5936ef)

![Screen Shot 2020-01-10 at 12.58.16 PM.png](https://images.zenhubusercontent.com/5caf676becad11531cc417cb/bd38cc06-5abc-414a-b3cd-6116a92ebe5e)

**Comment by johnclary** [comment]
@dianamartin ready for testing :)

**Comment by dianamartin** [comment]
I used WO `WRK20-073355`. The Inventory Request is currently `Needs AIMS Entry`. 
I added a new inventory request so it would show up on the Finance site as `New Request`. 

On the Finance site, I saw that I could `Edit` the inventory request item. I changed the quantity from 3 to 2 and added commentary to the comment box. 

Originally 
![image](https://user-images.githubusercontent.com/21112975/72182935-33887e80-33b2-11ea-8a93-306650b21b75.png)

Updated to
![image](https://user-images.githubusercontent.com/21112975/72182985-5155e380-33b2-11ea-95cc-a7d1926ebacd.png)
